api/src/test/java/com/cloud/agent/api/storage/OVFHelperTest.java

```diff
@@ -270,9 +270,9 @@ public class OVFHelperTest {
 
     private String eulaSections =
             "<VirtualSystem xmlns=\"http://schemas.dmtf.org/ovf/envelope/1\" xmlns:cim=\"http://schemas.dmtf.org/wbem/wscim/1/common\" xmlns:ovf=\"http://schemas.dmtf.org/ovf/envelope/1\" xmlns:rasd=\"http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_ResourceAllocationSettingData\" xmlns:vmw=\"http://www.vmware.com/schema/ovf\" xmlns:vssd=\"http://schemas.dmtf.org/wbem/wscim/1/cim-schema/2/CIM_VirtualSystemSettingData\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\">\n" +
-            "<EulaSection>\n" +
-            "      <Info>end-user license agreement</Info>\n" +
-            "      <License>END USER LICENSE AGREEMENT\n" +
+            "<ovf:EulaSection>\n" +
```

Comment:
If it can be either `<EulaSection>` or `<ovf:EulaSection>` should we have a separate test?

Reply:
in fact, it could be `<any string>:EulaSection`, but I guess that one is covered. I agree we should add the non-prefixed case in the tests somehow.

Reply:
In the same string there are two section,  <EulaSection> and <ovf:EulaSection>. So it covers both the cases.
Anyways I've added two separate test methods.